src/subnets.json

```diff
@@ -150,8 +150,8 @@
         "description": ""
     },
     "30": {
-        "name": "WomboAI",
-        "github": "https://github.com/womboai/wombo-bittensor-subnet",
+        "name": "",
```

Comment:
Should be "Unknown"

Reply:
"Unknown" implies - "there's a subnet here - we just don't know what it is yet."  In this case - there is no SN at 30.